lib/uri_parser.js

```diff
@@ -197,10 +197,12 @@ const CASE_TRANSLATION = {
   serverselectiontimeoutms: 'serverSelectionTimeoutMS',
   serverselectiontryonce: 'serverSelectionTryOnce',
   heartbeatfrequencyms: 'heartbeatFrequencyMS',
-  appname: 'appName',
```

Comment:
The spec specifies that `appname` is all lowercase. This change didn't break any tests, but do we think that users are depending on having `appName`?

Reply:
if we're going to be non-spec-compliant on this, we should make sure it has a deprecation message.

Reply:
that sounds reasonable (re: deprecation)

Reply:
This change makes us spec compliant. The spec says that `appname` should be lowercase, not camelcase.

Reply:
Eh, how we represent it internally doesn't matter. I think it will be fine.

Reply:
@daprahamian can we resolve this?